src/App/Header/Mainmenu/Mainmenu.js

```diff
@@ -7,16 +8,16 @@ function Mainmenu() {
     <div className='menu'>
       <ul>
         <li>
-          <Link to='/'>Home</Link>
+          <Link to={routes.HOME}>Home</Link>
         </li>
         <li>
-          <Link to='/products'>Product</Link>
+          <Link to={routes.PRODUCTS}>Product</Link>
```

Comment:
Here was another page. Before your change pattern was `/products`, now it's `/products/:id`